CONTRIBUTING.md

```diff
@@ -103,3 +102,58 @@ developers.
 
   More information about tool development can be found [on the
   wiki](https://wiki.galaxyproject.org/Develop).
+
+## Handling Pull Requests
+
+Everyone is encouraged to express opinions and issue non-binding votes on pull
+requests, but only members of the *committers* group may issue binding votes
+on pull requests. Information on the *committers* group can be found in the
+[organization document](https://github.com/galaxyproject/galaxy/blob/dev/doc/source/project/organization.rst)
+describing governance of the core Galaxy code base.
+
+Votes on pull requests should take the form of
+[+1, 0, -1, and fractions](http://www.apache.org/foundation/voting.html)
+as outlined by the Apache Foundation.
+
+Pull requests modifying pre-existing releases should be restricted to bug fixes
+and require at least 2 *+1* binding votes from someone other than the author of
+the pull request with no *-1* binding votes.
+
+Any pull request changing or clarifying the procedures governing this
+repository must remain open for at least 4 days and require at least 3 *+1*
```

Comment:
@blankenberg We had everyone in a room voting and I don't think we had 10 truly positive votes for the uncontroversial rule changes (e.g. adding additional committers). I would argue we need agile governance for all the same reasons we should have agile code - so that we can adapt to changing situations.

Lazy consensus should win the day especially given the extraordinary power we are given each committer with the insurmountable -1 veto.